governance.md

```diff
@@ -171,7 +171,7 @@ All contributors must sign the CNCF CLA, as described [here](CLA.md).
 [owners]: community-membership.md#subproject-owner
 [sig charter process]: committee-steering/governance/README.md
 [short template]: committee-steering/governance/sig-governance-template-short.md
-[kubernetes repository guidelines]: kubernetes-repositories.md
+[kubernetes repository guidelines]: github-management/kubernetes-repositories.md
```

Comment:
Can you add a leading slash to all links here (`/github-management/kubernetes-repositories.md`) so that they stay intact even if docs move?

Reply:
Sure